lib/cn/kvset.h

```diff
@@ -83,6 +83,36 @@ enum {
     KVSET_MISS_KEY_TOO_LARGE = -2,
 };
 
+/**
+ * struct kvset_vgroup_map - vgroup map
+ *
+ * A vgroup map stores the number of vgroups and its corresponding index mappings
```

Comment:
Suggest more background at the start of the comment:

Vblock indexes are stored with keys in a kvset's kblocks and are used to identify which vblock in the kvset's list of vblocks holds a key's value.   A vgroup map is associated with a kvset and is used to convert these indexes so they reference the correct vblock.  This conversion is only necessary with kvsets that have been k-compacted because k-compaction changes the vblock list but does not update the vblock indexes stored in the kblocks.

Reply:
Sure, will add. Thanks! I think you meant to write kvset split wherever k-compaction is referenced here.

Reply:
Of course I meant that. :)